docs/production/db-creds-rotation.md

```diff
@@ -0,0 +1,116 @@
+# DB credentials rotation
+
+Postgres has a reasonably challenging way to rotate credentials. See:
+
+- http://davidhollenberger.com/2017/03/16/postgres-credential-rotation/
+
+## Setup
+
+_This has already been setup in the production DB._
+
+Our default user is the `postgres` user - this user/role owns all the tables, so we
+need the other roles to be able to act as the `postgres` user, so we'll add two roles
+that we can rotate between.
+
+- Add two users who can't login, `user1` and `user2`:
+
+```
+create role user1 with nologin in role postgres;
+create role user2 with nologin in role postgres;
+```
+
+- Set their default login role as `postgres`
+
+```
+alter role user1 set role postgres;
+alter role user2 set role postgres;
+```
+
+## Overview of pw rotation
+
+- existing infra will be using either `user1` or `user2`. Find out which one.
```

Comment:
I don't understand. We just added these 2 users - how are they 'already' being used?

Reply:
This applies to subsequent migrations (during the first migration, the postgres user is in use)